CHANGELOG.md

```diff
@@ -53,51 +53,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for Validator configurable threshold (<https://github.com/openvinotoolkit/datumaro/pull/250>)
 
 ### Changed
-- LabelMe format saves dataset items with their relative paths by subsets without changing names (<https://github.com/openvinotoolkit/datumaro/pull/200>)
-- Allowed arbitrary subset count and names in classification and detection splitters (<https://github.com/openvinotoolkit/datumaro/pull/207>)
+- LabelMe format saves dataset items with their relative paths by subsets
+without changing names (<https://github.com/openvinotoolkit/datumaro/pull/200>)
```

Comment:
Please align continuation lines with the first line, e.g.:

```suggestion
- LabelMe format saves dataset items with their relative paths by subsets
  without changing names (<https://github.com/openvinotoolkit/datumaro/pull/200>)
```

Reply:
Done